src/core-layers/path-layer/path-layer-vertex.glsl.js

```diff
@@ -68,8 +68,10 @@ vec3 lineJoin(vec3 prevPoint, vec3 currPoint, vec3 nextPoint) {
 
   float lenA = length(deltaA);
   float lenB = length(deltaB);
-  vec2 dirA = lenA > 0. ? normalize(deltaA) : vec2(1.0, 0.0);
-  vec2 dirB = lenB > 0. ? normalize(deltaB) : vec2(1.0, 0.0);
+  lenA = lenA > 0.1 ? lenA : 0.0;
+  lenB = lenB > 0.1 ? lenB : 0.0;
+  vec2 dirA = lenA > 0. ? normalize(deltaA) : vec2(0.0, 0.0);
```

Comment:
Nit: why not change the this condition from `lenA > 0.` to `lenA > 0.1` and remove above modification of `lenA` (similarly for `lenB`), or is this is the workaround to avoid some GPU behaviors? As above, a comment would be great.

Reply:
lenA and lenB are used on code below, not only here.